tezos/encoding/src/de.rs

```diff
@@ -397,7 +397,7 @@ impl<'de> de::MapAccess<'de> for StructDeserializer<'de> {
                 let (ref field, ref value) = *item;
                 self.value = Some(value);
                 seed.deserialize(StringDeserializer {
-                    input: field.clone(),
+                    input: String::from(field.as_str()), // TODO: check if custom deserializer for FieldName is appropriate
```

Comment:
@rspendl 
what about this TODO?

Reply:
Oops, this is a leftover comment, no need for custom deserializer. I'll delete that.